google/api_core/exceptions.py

```diff
@@ -34,6 +35,14 @@
 _HTTP_CODE_TO_EXCEPTION = {}
 _GRPC_CODE_TO_EXCEPTION = {}
 
+# Additional lookup table to map integer status codes to grpc status code
+# grpc does not currently support initializing enums from ints
+# i.e., grpc.StatusCode(5) raises an error
+if grpc is not None:  # pragma: no branch
+    _INT_TO_GRPC_CODE = {}
```

Comment:
Move this up to unconditional module scope? (to address @crwilcox comment below).